atlp-rwanda/Avengers-Phantom-FE
# Assingning bus to route

## What does this PR do?
I've adding bus to route assignment in phantom project

## Description of Task to be completed?
These changes allow only administrator and operator to assign buses to routes
- [x] list of buses
- [x] List of routes
- [x] Assignment
- [x] Un-assignment 
 
## How should this be manually tested?
This includes only admin || operators authentication.  That is only when they can select bus and route to be assigned on and assign them.

step1: clone git repo if not yet done
step2: cd Avengers-Phantom-FE
step3: npm i or npm install to install dependencies
step4: `npm run dev` to start the server
step5: switch to` assigning_bus_to_route` branch
step6: Login to go to the dashboard with administrator or operator user accounts to check

## Screenshot

![image](https://user-images.githubusercontent.com/77101006/177943445-db3c82d3-5d76-42af-8e83-fc0cd3bf9d53.png)
